### PR TITLE
Updated tour for Accumulo 2.0

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

File: tour/authorizations-code.md [hunk 1]
```diff
@@ -6,50 +6,56 @@ Below is a solution for the exercise.
...
 
-    // Read and print all rows of the commissioner can see. Pass Scanner proper authorizations
-    Connector commishConn = mac.getConnector("commissioner", "gordonrocks");
-    try (Scanner scan = commishConn.createScanner("GothamPD", auths)) {
-        System.out.println("Gotham Police Department Persons of Interest:");
-        for (Map.Entry<Key, Value> entry : scan) {
-            System.out.printf("Key : %-60s  Value : %s\n", entry.getKey(), entry.getValue());
-        }
+  // Create a client as the 'commissioner' user
+  try (AccumuloClient client = Accumulo.newClient().from(mac.getClientProperties())
```
Comment (keith-turner) [inline tour/authorizations-code.md]: Could do 

```java
try (AccumuloClient client = Accumulo.newClient().from(mac.getClientProperties()).as("commissioner", "gordonrocks").build(); 
      Scanner scan = client.createScanner("GothamPD")){
   //.....
}
```

File: tour/data-model.md [hunk 2]
```diff
@@ -17,12 +17,12 @@ The **value** is where the actual data is stored. For brevity, we often refer to
 Take a closer look at the Mutation object created in the first exercise:
 ```java
 Mutation mutation = new Mutation("id0001");
-mutation.put("hero","alias", "Batman");
+mutation.at().family("hero").qualifier("alias").put("Batman");
```
Comment (ctubbsii) [inline tour/data-model.md]: I don't know "at" is the right verb here. Perhaps "of" seems more sensible? It's a mutation "of" a family/qualifier, not "at" a family/qualifier.

Comment (mikewalch) [reply tour/data-model.md]: I would create an issue for that.  I am just using the current API.

Comment (ctubbsii) [reply tour/data-model.md]: The tour, and our other example code allow us to exercise and question the usability of our API. The question here is whether "at" is grammatically appropriate, in the context of this specific example. If yes, then I won't do anything. If no, then I'll create a follow-on ticket.

Comment (keith-turner) [reply tour/data-model.md]: I think the the issue where this name was discussed `at` was used thinking that the family and qualifier are like coordinates/positions/locations within a row.  It seems like there was a lot of discussion on the issue about this name.  Personally I like the name.

Comment (ctubbsii) [reply tour/data-model.md]: Thanks. I agree `at` makes more sense with `put`. `of` might make more sense with `set`, but I'm fine with how it is.

**Comment by milleruntime** [comment]
@mikewalch are you planning on working on this?

**Comment by mikewalch** [comment]
> @mikewalch are you planning on working on this?

Feel free to pick it up or close it.

**Pull request closed without merging**